src/app/users/[username]/page.tsx

```diff
@@ -11,40 +17,104 @@ type PropTypes = {
-    const userProfile = await prisma.user.findUnique({
+    const profile = await readUserProfile({username})
+
+    if (!profile) {
+        notFound()
+    }
+
+    const groupIds = profile.memberships.map(group => group.groupId)
+    const committees = await readCommitteesFromIds(groupIds)
+
+    const studyProgramme = await prismaCall(() => prisma.studyProgramme.findFirst({
         where: {
-            username
+            id: {
+                in: groupIds
+            }
+        },
+        include: {
+            group: {
+                include: {
+                    memberships: {
+                        include: {
+                            omegaOrder: {
+                                select: {
+                                    order: true
+                                }
+                            }
+                        }
+                    }
+                }
+            }
         }
-    })
+    }))
 
-    if (!userProfile) {
-        notFound()
+    if (!studyProgramme) {
+        throw new Error("studyProgramme not found")
+    }
+
+    const order = studyProgramme.group.memberships[0].omegaOrder.order
+
+    let profileImage = profile.image
+    if (!profileImage) {
+        profileImage = await readSpecialImage("DEFAULT_PROFILE_IMAGE")
     }
 
     return (
-        <>
-            <h1>{`${user.firstname} ${user.lastname}`}</h1>
-            <p>{`E-post: '${user.email}'`}</p>
-            <p>{`Bruker-ID: ${user.id}`}</p>
-            <h2>Tillganger:</h2>
-            <ul>
-                {me && permissions.map(permission => <li key={uuid()}>{permission}</li>)}
-            </ul>
-            <h2>Grupper:</h2>
-            <ul>
-                {me && memberships.map(membership => <li key={uuid()}>{membership.groupId}</li>)}
-            </ul>
-            {me && <Link href="/logout">Logg ut</Link>}
-        </>
+        <div className={styles.pageWrapper}>
+            <div className={`${styles.top} ${styles.standard}`}> {/* TODO change style based on flair */}
+            </div>
+            <div className={styles.profileContent}>
+                <div className={styles.imageWrapper}>
+                    <Image className={styles.profilePicture} image={profileImage} width={240}/>
+                </div>
+                <div className={styles.header}>
+                    <h1>{`${profile.firstname} ${profile.lastname}`}</h1>
+                    <p className={styles.studyProgramme}>{studyProgramme.name} {`(${studyProgramme.code})`}</p>
+                    <div className={styles.committeesWrapper}>
+                        {committees.map(committee => <div className={styles.committee}><p>{committee.name}</p></div>)} {/* TODO change to your own committee title instead of committee name*/}
+                    </div>
+                    <hr/>
+                    <p className={styles.orderText}>{profile.sex == "FEMALE" ? "Syster" : "Broder"} uudaf {order}´dis orden i Sanctus Omega Broderskab</p>
```

Comment:
Always use tripple equals in javascript. I think this is currently one of the linter errors for this branch. Also, I think a better approach would be to use a mapping something like this:

```ts
const MEMBERSHIP_RELATION = {
    FEMALE: 'Syster',
    MALE: 'Broder',
    OTHER: 'Soesken',
} satisfies Record<SEX, string> // Ensures that all possible SEX values exists as keys in the object
```

So that you could then do:
```ts
{MEMBERSHIP_RELATION[profile.sex ?? 'OTHER']} uudaf {order}´dis orden i Sanctus Omega Broderskab
```

Reply:
Changed my mind. I put it in users/ConfigVars